meson.build

```diff
@@ -64,7 +64,7 @@ pkg_conf_data.set('GTKMM_PLPLOT_MAJOR_VERSION', gtkmm_plplot_major_version)
 pkg_conf_data.set('GTKMM_PLPLOT_MINOR_VERSION', gtkmm_plplot_minor_version)
 
 # Dependencies
-gtkmm_dep = dependency('gtkmm-3.0', version: '>=3.0.0')
+gtkmm_dep = dependency('gtkmm-4.0', version: '>=4.0.0')
 plplotcxx_dep = dependency('plplot-c++', version: '>= 5.11.0')
 plplot_dep = dependency('plplot', version: '>= 5.11.0')
 pkg_conf_data.set('PLPLOT_CXX', 'plplot-c++ >= 5.11.0')
```

Comment:
I think that after bumping the minimum required version of plplot to 5.11.0, we no longer need to use `GTKMM_PLPLOT_PLPLOT_5_11_0` anywhere in the code